packages/fluentui/react-northstar/src/components/Segment/Segment.tsx

```diff
@@ -23,51 +32,86 @@ export interface SegmentProps
   /**
    * Accessibility behavior if overridden by the user.
    */
-  accessibility?: Accessibility;
+  accessibility?: Accessibility<never>;
 
   /** An segment can show it is currently unable to be interacted with. */
   disabled?: boolean;
 
   /** A segment can have its colors inverted for contrast. */
   inverted?: boolean;
+
+  rtlAttributes?: typeof rtlTextContainer.getAttributes;
```

Comment:
Please why do you need this new prop? I don't see where we use it anywhere in repo

Reply:
Only added here because it was on the proptypes.... Will remove from both then